public/app/features/dashboard/state/getPanelPluginToMigrateTo.ts

```diff
@@ -7,12 +7,30 @@ export function getPanelPluginToMigrateTo(panel: any, forceMigration?: boolean):
     return autoMigrateRemovedPanelPlugins[panel.type];
   }
 
+  const isUrlFeatureFlagEnabled = (featureName: string) => {
+    const flag = '__feature.' + featureName;
+
+    const urlParams = new URLSearchParams(window.location.search);
+    const featureFlagValue = urlParams.get(flag);
+
+    return featureFlagValue === 'true' || featureFlagValue === '';
+  };
```

Comment:
Won't it be easier to allow enabling some of the feature flags via URL parameters in `grafana-runtime/src/config.ts` like we did before? This way we don't have to re-parse the URL here and repeat the checks below:

```ts
  if (
    panel.type === 'graph' &&
    (shouldMigrateAllAngularPanels ||
      config.featureToggles.autoMigrateGraphPanel ||
      isUrlFeatureFlagEnabled('autoMigrateGraphPanel'))
  )
```

:point_down: 

```ts
  if (
    panel.type === 'graph' &&
    (shouldMigrateAllAngularPanels ||
      config.featureToggles.autoMigrateGraphPanel
  )
```

Reply:
I know what you mean, but this is supposed to be temporary - the idea of this approach is to be able to migrate and test specific dashboards (the ones for which you click "Try migration" ) and that would not be possible to do if we enable the feature flag - the feature flag is enabled globally and if the user navigates away from the dashboard where tried the migration, other dashboards would be migrated too, without notice or possibility to check the previous state. This is mainly why we moved away from that approach..

Reply:
> if the user navigates away from the dashboard where tried the migration, other dashboards would be migrated too, without notice or possibility to check the previous state.

and the dashboard they navigate to may also be broken by the now-enabled auto migration. they have to understand that to "fix" it they have to refresh browser, ehich is super confusing.

Reply:
Ah! Right, I totally forgot about that :sweat: 

Sounds good then! :+1: